gridfinity-rebuilt-holes.scad

```diff
@@ -278,7 +287,19 @@ module block_base_hole(hole_options, o=0) {
                 if(crush_ribs) {
                     ribbed_cylinder(magnet_radius, magnet_inner_radius, magnet_depth, MAGNET_HOLE_CRUSH_RIB_COUNT);
                 } else {
-                    cylinder(h = magnet_depth, r=magnet_radius);
+                    if(embedded) {
+                      // Leave a gap for the screwdriver for recycling
+                      intersection() {
+                        translate([-magnet_radius,-magnet_radius * 0.15,0])
+                        cube([2*magnet_radius, magnet_radius * 0.3, embedded_depth]);
+                        cylinder(h = embedded_depth, r=magnet_radius);
+                      }
```

Comment:
Given this is inside a `difference()`, have you confirmed if this works correctly with other options?  If it is not supposed to be used with anything else, then you will need to add checks similar to those which currently exist for refined_hole.